### ISSUE TITLE
Getting dovecot error while running this ansible script

### ISSUE THREAD

**Issue by sorcer1122** [progmaticltd/homebox#150]
Hello!

I have managed to fix all the issues with dovecot by reverting to default settings. However, I am still getting the following error with dovecot, after which the script stops.

Posting it here for benefit of others.

RUNNING HANDLER [dovecot : Restart dovecot] **********************************************************************************************************************************************
task path: /home/user/homebox/install/playbooks/roles/dovecot/handlers/main.yml:2
fatal: [homebox]: FAILED! => changed=false 
  msg: |-
    Unable to restart service dovecot: Job for dovecot.service failed because the control process exited with error code.
    See "systemctl status dovecot.service" and "journalctl -xe" for details.

**Comment by krimaeus** [comment]
Well, do what it says. Run `systemctl status dovecot` and/or `journalctl -xe`, and see if you can figure out what the problem is. Otherwise post the logs here.

**Comment by arodier** [comment]
@sorcer1122, I believe the error you have is related to the fact you have previously tried to override the Dovecot configuration.
You can try to fix it, by posting here what @krimaeus mentioned, or you can also restart an installation from scratch.
Good luck.

**Comment by sorcer1122** [comment]
OK, I reinstalled OS and started from scratch. Got the same error, run systemctl status dovecot and guess what I saw there....

 doveconf: Fatal: Error in configuration file /etc/dovecot/conf.d/10-auth.conf line 41: Unknown setting: username_chars
Jun 12 17:40:04 coldmail.cc dovecot[5236]: doveconf: Error: managesieve-login: dump-capability process returned 89
Jun 12 17:40:04 coldmail.cc dovecot[5236]: doveconf: Fatal: Error in configuration file /etc/dovecot/conf.d/10-auth.conf line 41: Unknown setting: username_chars

This is because I have modified 10-auth.conf trying to fix username_chars error.

Wisdom: make sure you remember what you modify and revert it back!
